content/doc/developer/security/index.adoc

```diff
@@ -96,6 +98,63 @@ To do this, write Jelly like this:
 NOTE: This is not to be confused with the +checkPermission+ invocation in your operation.
 Users can still hit the URL directly, so you still need to protect the operation itself, in addition to disabling the UI rendering
 
+=== Authentication methods
+
+In Jenkins the security engine that is used is AcegiSecurity (ancestor of SpringSecurity).
+Without any special plugins to manage authentication, an instance of Jenkins is packaged
+with the following authentication methods:
+
+* Web UI
+** When you use the form on the login page, using the fields j_username and j_password
```

Comment:
```adoc
** When you use the form on the login page, using the fields +j_username+ and +j_password+
```

Reply:
corrected, same for the '--username' or '-auth' options